content/en/docs/attestation/client-tool/_index.md

```diff
@@ -1,16 +1,75 @@
 ---
 title: KBS Client Tool
 description: Simple tool to test or configure Key Broker Service and Attestation Service
-weight: 4
+weight: 20
 categories:
 - docs
 tags:
 - docs
 - kbs
 ---
 
-This is a simple client for the Key Broker Client (KBS) that facilitates testing of the KBS and other basic attestation flows.
+Trustee can be configured using the KBS Client tool.
+Other parts of this documentation may assume that this tool is installed.
 
-You can run this tool inside of a TEE to make a request with real attestation evidence. You can also provide pre-existing evidence or use the sample attester as a fallback.
+### Install
 
-The client tool can also be used to provision the KBS/AS with resources and policies.
+The KBS Client tool can be installed in two ways.
+
+#### With ORAS
+
+Pull the KBS Client with ORAS.
+
+```bash
+oras pull ghcr.io/confidential-containers/staged-images/kbs-client:latest
+chmod +x kbs-client
+```
+
+#### From source
+
+Clone the Trustee repo.
+```bash
+git clone https://github.com/confidential-containers/trustee.git
+```
+
+Build the client
+```
+cd kbs
+make CLI_FEATURES=sample_only cli
+sudo make install cli
+```
+
+The `sample_only` feature is used to avoid building hardware attesters into the KBS Client.
+In addition to configuring the KBS, the client can also be used inside of an enclave
+to retrieve resources.
+In this case the attester for that enclave must be built.
+You can remove the `sample_only` feature to build all the attesters.
+This will require extra dependencies.
```

Comment:
The comments about attesters could be relevant for the ORAS based setup too so perhaps some of this could be moved to the intro. The ORAS section could also mention that the pre-built `kbs-client` has no attester support either.

Reply:
+1.  It's important to clarify that the published kbs-client has only sample attester support.
May be as a next step we can create a package with all coco tools in one place to make it easier for users to try out without building the tools.

I'm doing something like this here - https://github.com/confidential-devhub/coco-tools/blob/main/Containerfile

Reply:
I added some more info about the feature to the intro section to clarify. This does reveal that our tools aren't quite as easy to consume as they should be.

Reply:
> This does reveal that our tools aren't quite as easy to consume as they should be.
Yes.

Reply:
> This does reveal that our tools aren't quite as easy to consume as they should be.

FWIW, I'm planning to re-work `tdx-attester`  to be fully Rust based by default.